main.py

```diff
@@ -91,8 +96,11 @@ def main():
 
 		if rp_config.getboolean("WaitForUserInput", fallback=False):
 			input("Press enter to continue...")
-
+	
 	print("DONE!")
+	# End timer and display in hours, minutes and seconds.
+	time_obj = datetime.utcfromtimestamp(time.time() - start_time)
```

Comment:
Please move this so it gets printed out once per pass. It'll be more useful that way.

Reply:
How about I do both - a pass timer and an overall timer?

Reply:
The copy and remove timers always come up with 00:00:00 - thinking remove them and just have them for the comfyui, ebsynth_blend and alpha_blend passes?

Reply:
> How about I do both - a pass timer and an overall timer?

You don't need overall since the last pass will still print out how long it took. Unless you meant a different timer per pass?

> The copy and remove timers always come up with 00:00:00 - thinking remove them and just have them for the comfyui, ebsynth_blend and alpha_blend passes?

Printing out 00:00:00 is fine.

Reply:
I intpreted it as the pass timer as elpased time per pass rather than a cumulative time. PR updated.

BTW: I was half asleep when I did the commit this morning and committed way more than intended. I've backed it out.